client/src/stories/Component/Brand/Brand.js

```diff
@@ -0,0 +1,41 @@
+import React from "react";
+import PropTypes from "prop-types";
+import styled from "@emotion/styled";
+
+function Brand({ imgSrc, imgName, firmName }) {
+  return (
+    <Container>
+      <img src={imgSrc} alt={imgName} />
+      <p>{firmName}</p>
+    </Container>
+  );
+}
+
+export default Brand;
+
+//Proptypes
+
+Brand.propTypes = {
+  imgSrc: PropTypes.string,
+  imgName: PropTypes.string,
+  firmName: PropTypes.string,
+};
+
+// Styling
+
+const Container = styled.div`
+  display: grid;
+  grid-template-columns: 65px 1fr;
+  background-color: #3f5941;
+  p {
+    color: #f2f2f2;
```

Comment:
I think you could save your colors, among other things, as global variables! Down below I saw another example of this issue.

Reply:
And here above with the background-color too